gclient/tests/upload.rs

```diff
@@ -27,6 +27,8 @@ const PATHS: [&str; 2] = [
     "../target/wat-examples/inf_recursion.wasm",
 ];
 
+const PING_PATH: &str = "../target/wasm32-unknown-unknown/release/demo_ping.opt.wasm";
```

Comment:
I meant referencing the binary this way
```rust
demo_mul_by_const::WASM_BINARY
```

Reply:
> I meant referencing the binary this way instead of using path string
> 
> ```rust
> demo_mul_by_const::WASM_BINARY
> ```

This can only be done with `examples/binaries` as they have the `WASM_BINARY` constant.